style.css

```diff
@@ -702,3 +705,388 @@ h2 {
 }
 
 /* Contact Form end */
+
+/* ************************************************************************************** */
+
+/* ------------------------- Desktop Version ------------------- */
+@media screen and (min-width: 375px) {
+  header {
+    display: flex;
+    width: 100vw;
+    height: 916px;
+    border-bottom-left-radius: 100px;
+  }
+
```

Comment:
- [ ] As we are using the mobile-first approach, I think it would be nice to use the @media query suggested by the requirements  `@media screen and (min-width: 990px)` or  `@media screen and (min-width: 1080px)` or between the two.

- [ ] Currently, your mobile version is broken. Note that even if you are building the desktop version the mobile version should not remain the same. Kindly I would suggest you repair the mobile version.